[2025-07-01 07:05:04] Traceback (most recent call last):
[2025-07-01 07:05:04]   File "/app/bot.py", line 25, in <module>
[2025-07-01 07:05:04]     from cogs.draft_prob import DraftLotteryCog
[2025-07-01 07:05:04]   File "/app/cogs/draft_prob.py", line 1570
[2025-07-01 07:05:04]     "
[2025-07-01 07:05:04]     ^
[2025-07-01 07:05:04] SyntaxError: unterminated string literal (detected at line 1570)
[2025-07-01 07:05:03] ERROR failed health checks after 7 attempts with error Readiness probe failed: dial tcp 10.244.21.38:8080: connect: connection refused
[2025-07-01 07:05:34] ERROR component terminated with non-zero exit code: 1,
[]
